script.js

```diff
@@ -1,4 +1,51 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var computerNumber = randomNumber();
+  var computerChoice = "";
+  console.log("compter number is supposed to be " + computerNumber);
+  var lowercase = input.toLowerCase();
+  console.log("Uppercase and Lowercase does not matter");
+
+  if (computerNumber == 1) {
+    computerChoice = "scissors";
+  }
+
+  if (computerNumber == 2) {
+    computerChoice = "paper";
+  }
+
+  if (computerNumber == 3) {
+    computerChoice = "stone";
+  }
+
+  var myOutputValue =
+    "The computer chose " +
+    computerChoice +
+    ". <br> You chose " +
+    lowercase +
+    "<br><br> You lose! Bummer." +
+    "<br><br> Hey, dont stop here keep going! Try again.";
```

Comment:
Good on creating `myOutputValue` and assigning it to the string containing the copywriting for the loss condition.

You have completed the [input validation step](https://basics.rocketacademy.co/projects/project-1-scissors-paper-stone/project-1-scissors-paper-stone-part-1#input-validation) of [Scissors, Paper, Stone Part 1's Base](https://basics.rocketacademy.co/projects/project-1-scissors-paper-stone/project-1-scissors-paper-stone-part-1). The only requirement left for Part 1 Base's is the [determination of the win and draw conditions](https://basics.rocketacademy.co/projects/project-1-scissors-paper-stone/project-1-scissors-paper-stone-part-1#basic-scissors-paper-stone). You already have the loss condition as the default, as defined by [lines 20 to 26](https://github.com/rocketacademy/basics-scissors-paper-stone/blob/fa3bd1c0eeadf79926d0ef47e83442e0c0326862/script.js#L20-L26). Thus, you only need to check for wins and draws next.

But don't be in a hurry to complete this yet! You can attempt to try to finish this at your own pace once the course is complete. I will just drop some hints for you here to try on your own next time.

The winning checks involves the following pseudocode:
```
// The player wins if
// (1) (The player selects "paper", AND the computer selects "stone"), OR
// (2) (The player selects "stone", AND the computer selects "scissors"), OR
// (3) (The player selects "scissors", AND the computer selects "paper")
```
which translate to the following JavaScript code:
```
if (
  (lowercase == "paper" && computerChoice == "stone") ||
  (lowercase == "stone" && computerChoice == "scissors") ||
  (lowercase == "scissors" && computerChoice == "paper")
) {
  ...
}
```

The draw condition is much simpler, in pseudocode:
```
// The player draws if his selection is the same as the computer's
// which means lowercase is the same (equivalent) to computerChoice
```
which translate to this in JavaScript:
```
if (lowercase == computerChoice) {
  ...
}
```